knife/lib/chef/knife/core/gem_glob_loader.rb

```diff
@@ -45,22 +45,13 @@ def gem_and_builtin_subcommands
           find_subcommands_via_dirglob
         end
 
-        def find_subcommands_via_dirglob
```

Comment:
This was removed because it already exists in `SubcommandLoader` with nearly the same behavior - base class excludes chef/knife/version.rb from the list of commands.